Create ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md (#2648)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md

---
date: ""

> title: Prevent duplicate entries in navigation when configuring top level nodes

Reviewer: ```suggestion
title: Prevented duplicate entries in the navigator when configuring top level nodes
```

> title: Prevented duplicate entries in the navigator when configuring top level nodes

Reviewer: ```suggestion
title: Prevented duplicate asset entries in the navigator when configuring top level nodes
```

> title: Fix duplicate navigation entries when configuring cockpit root nodes (#7412) [GRAFT][release/cd] (#7584)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix duplicate navigation entries when configuring cockpit root nodes
```

> title: Fix duplicate navigation entries when configuring cockpit root nodes (#7412) [GRAFT][release/cd] (#7584)

Reviewer 2: ```suggestion
title: Prevent duplicate entries in navigation when configuring top level nodes
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:

>   - value: component-YbYJ3gLU_

Reviewer: ```suggestion
  - value: component-YdSEScrEC
```

>     label: Web SDK

Reviewer: ```suggestion
    label: Cockpit
```

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61026
version: 1021.4.1
---

> Fix duplicate navigation entries when configuring cockpit root nodes (#7412) [GRAFT][release/cd] (#7584)

Author: The Claude AI suggested to use the following body:
```suggestion
In certain scenarios, duplicate navigation entries could appear in the Cumulocity cockpit when configuring root nodes. This issue has been resolved, ensuring that navigation entries are now displayed correctly without any duplicates. Cockpit users will no longer encounter redundant entries, leading to a cleaner and more intuitive navigation experience.
```

> Fix duplicate navigation entries when configuring cockpit root nodes (#7412) [GRAFT][release/cd] (#7584)

Reviewer 2: ```suggestion
Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries in the main navigation menu. The navigator now properly updates without creating redundant menu items.
```

> Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries in the main navigation menu. The navigator now properly updates without creating redundant menu items.

Reviewer: @DawidSAG This is not yet clear. I assume you are talking about assets. Can you please add this info?

Reviewer 2: What about:

```suggestion
Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries of assets in the main navigation menu. The navigator now properly updates without creating redundant menu items.
```

> Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries of assets in the main navigation menu. The navigator now properly updates without creating redundant menu items.

Reviewer: ```suggestion
Fixed an issue where editing top level nodes in the **Application configuration** page created duplicate entries of assets in the navigator. Now the navigator updates properly without creating redundant menu items.
```

> Fixed an issue where editing top level nodes in the **Application configuration** page created duplicate entries of assets in the navigator. Now the navigator updates properly without creating redundant menu items.

Reviewer: ```suggestion
Fixed an issue where editing top level nodes in the **Application configuration** page in the Cockpit application created duplicate entries of assets in the navigator. Now the navigator updates properly without creating redundant menu items.
```
